modules/monitoring-support-considerations.adoc

```diff
@@ -28,6 +28,8 @@ This procedure is a supported exception to the preceding statement.
 * *Modifying resources of the stack.* The {product-title} monitoring stack ensures its resources are always in the state it expects them to be. If they are modified, the stack will reset them.
 * *Deploying user-defined workloads to `openshift-&#42;`, and `kube-&#42;` projects.* These projects are reserved for Red Hat provided components and they should not be used for user-defined workloads.
 * *Enabling symptom based monitoring by using the `Probe` custom resource definition (CRD) in Prometheus Operator.*
+* *Manually deploying monitoring resources into namespaces with the `openshift.io/cluster-monitoring` label.* 
```

Comment:
I think we should more explicit,
`openshift.io/cluster-monitoring: "true"`
since namespace with `openshift.io/cluster-monitoring: "false"` and `openshift.io/cluster-monitoring: ""` is monitored by user workload monitoring

Reply:
You're 100% correct, I missed that in my review.